ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -198,6 +198,7 @@ export const NavBar = () => {
     </div>
     <Breadcrumb/>
     {window.location.pathname !== '/access-renewal' && <AccessRenewalNotificationMaybe/>}
+    {window.location.pathname !== '/data-access-requirements' && <LoginGovIAL2NotificationMaybe/>}
```

Comment:
Clicking the button on this notification takes you to `data-access-requirements` so we don't want to show it there.  Same pattern as `access-renewal`.